src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier{
+    Random random = new Random();
+    ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
The field 'colorSupplier' should also be private to follow encapsulation principles.

Reply:
The field 'colorSupplier' should also be private to follow encapsulation principles.